packages/explorable-explanations/src/lib/bubbles.js

```diff
@@ -0,0 +1,525 @@
+/*
```

Comment:
Moving this to module as it encapsulates functional unit of the application, responsible for a specific feature.